LibTest/js_interop/FunctionToJSExportedDartFunction/toJSCaptureThis_A02_t02.dart

```diff
@@ -20,15 +20,17 @@ import 'dart:js_interop';
 
 class C {}
 
-C? foo(pThis) {return null;}
+C? foo(JSObject pThis) {return null;}
 
-void bar(pThis, C c) {}
+void bar(JSObject pThis, C c) {}
```

Comment:
nit: It may be worth having a function with all valid return and parameter types except for the first parameter (the `this` value) to check that we still catch that.

Reply:
There is `void corge(pThis) {}` on the line 33.

Reply:
Ah I missed that, thanks!